File: plugin/src/org/jetbrains/yesod/lucius/LuciusLanguage.java
package org.jetbrains.yesod.lucius;

/**
 * @author Leyla H
 */

import com.intellij.lang.Language;

public class LuciusLanguage extends Language {
    public static final LuciusLanguage INSTANCE = new LuciusLanguage();

    public LuciusLanguage() {
        super("Lucius", "text/lucius");
    }
}

File: plugin/src/org/jetbrains/yesod/lucius/LuciusLanguage.kt
package org.jetbrains.yesod.lucius

/**
 * @author Leyla H
 */

import com.intellij.lang.Language

class LuciusLanguage : Language("Lucius", "text/lucius") {
    companion object {
        val INSTANCE: LuciusLanguage = LuciusLanguage()
    }
}